panel/chat/langchain.py

```diff
@@ -36,7 +39,7 @@ def __init__(
         self,
         instance: ChatFeed | ChatInterface,
         user: str = "LangChain",
-        avatar: str = "🦜️",
+        avatar: str = DEFAULT_AVATARS["langchain"],
```

Comment:
This will enable me and other users to change the `DEFAULT_AVATARS` globally.

Reply:
Unless I'm confused it won't. The method definition is evaluated on import and won't re-evaluate when you change the avatar. It's still a good idea but users won't be able to override the default.